src/components/AppModal.vue

```diff
@@ -35,6 +31,11 @@ export default {
   },
   methods: {
     ...mapMutations({ closeAppModal: CLOSE_APP_MODAL }),
+    ...mapActions({ editFormState: 'editFormState' }),
+    handleForm() {
+      this.$store.dispatch(this.actionType);
+      this.editFormState();
+    },
```

Comment:
This is not used, you can add an eslint rule that warns you if there are unused properties
https://eslint.vuejs.org/rules/no-unused-properties.html#options